packages/main/src/themes/Label.css

```diff
@@ -50,6 +55,11 @@
 	font-size: 1.25rem;
 	font-weight: bold;
 	position: relative;
+	font-style: normal;
 	vertical-align: middle;
 	line-height: 0;
+}
+
+:host([required][show-colon]) .ui5-label-required-colon:after {
```

Comment:
I cant see the RTL equivalent, do we need margin-right: 2px in RTL?

Reply:
Done